README.md

```diff
@@ -28,9 +27,15 @@ which it's run:
 - `modprobe`
 - `gpg`
 
+### Prebuilt Binaries
+
+The easiest way to get `acbuild` is to download one of the releases from
```

Comment:
The easiest way to get `acbuild` is to download one of the [releases](https://github.com/appc/acbuild/releases) from Github.